pytest.ini

```diff
@@ -4,5 +4,8 @@ env =
 
 norecursedirs = venv/bin venv/lib venv/lib64 .*
 python_files = test/*.py tests.py
-addopts =--flake8 -rsx
+addopts =--cov=. --junitxml=./test-results/pytest/results.xml --cov-report=xml --cov-report=term --flake8 -rsx
```

Comment:
Added a `.coveragerc` file to keep the test coverage requirements, but it seems the `--cov=.` flag here is overriding the `source` option in the rc file. I think if you just change this to `--cov` it would work.

Reply:
ah, true.